dist/images/Dockerfile.ubuntu

```diff
@@ -8,11 +8,11 @@
 #
 # So this file will change over time.
 
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 USER root
 
-RUN apt-get update && apt-get install -y arping iproute2 curl software-properties-common setpriv
+RUN apt-get update && apt-get install -y arping iproute2 curl software-properties-common # setpriv
```

Comment:
@roytman is `setpriv` part of ubuntu:20.04 by default? Why is it commented above? I thought we needed it for ubuntu based images

```
>git show 2b3c84b3bbfa25b2802d195c6230c1d085827aaf
commit 2b3c84b3bbfa25b2802d195c6230c1d085827aaf
Author: Girish Moodalbail <gmoodalbail@nvidia.com>
Date:   Tue Apr 16 14:36:39 2019 -0700

    dockerfile: ubuntu image needs setpriv package to run OVS daemons
    
    Signed-off-by: Girish Moodalbail <gmoodalbail@nvidia.com>
```

Reply:
Now it is part of `util-linux`, added it.
```
setpriv  -V
setpriv from util-linux 2.34
```